4337/package-lock.json

```diff
@@ -160,9 +160,9 @@
       }
     },
     "node_modules/@eslint/eslintrc": {
-      "version": "2.1.2",
```

Comment:
Are the version bumps intended?

Reply:
Jein. I was dealing with some bizzare issues, which were fixed by `rm -rf node_modules && rm -rf package-lock.json && npm i`